yb-voyager/src/utils/utils.go

```diff
@@ -983,3 +983,22 @@ func GenerateAnonymisationSalt(n int) (string, error) {
 	// convert bytes to hex to make it printable ASCII string
 	return hex.EncodeToString(b), nil
 }
+
+// IsSetEqual checks if two string slices contain the same elements regardless of order.
+// It sorts both slices and compares them for equality.
+func IsSetEqual(a, b []string) bool {
+	if len(a) != len(b) {
```

Comment:
you can just use `mapset.NewThreadUnsafeSet[string]` and then `a.Equal(b)` should work here